src/Controller/OcrController.php

```diff
@@ -36,13 +38,19 @@ class OcrController extends AbstractController
      * @param string $endpoint
      * @param string $key
```

Comment:
Docblock needs updating following the change to the signature

Reply:
Done.